components/grid/overview.md

```diff
@@ -144,13 +144,13 @@ The Grid supports custom content in various parts of the component such as data
 
 ## More Blazor Grid Features
 
-* [Selection]({%slug components/grid/selection/overview%}) - select one or multiple rows via clicks or checkboxes
-* [State]({%slug grid-state%}) - get or set the Grid configuration programmatically
-* [Toolbar]({%slug components/grid/features/toolbar%}) - define user actions in a toolbar above the header cells
-* [Hierarchy]({%slug components/grid/features/hierarchy%}) - nest Grids and visualize parent-child relations between data records
-* [Drag and drop rows]({%slug grid-drag-drop-overview%}) - move rows in a Grid or between different Grids
-* [Loading animation]({%slug grid-loading%}) - show a loading animation to improve user experience during long data operations
-* Scrolling - the Grid will show standard scrollbars automatically if the data does not fit the current component width and height.
+* [Selection]({%slug components/grid/selection/overview%})—select one or multiple rows through clicks or checkboxes or select one or multiple cells through clicks.
```

Comment:
As we are exposing Lasso selection, should we not mention it here?

Reply:
In the last retrospective meeting, we discussed that the "lasso" expression is not correct. The drag selection is described [here](https://github.com/telerik/blazor-docs/blob/773e39189878d7949d6e65a497b2a79a16bda6c6/components/grid/selection/cells.md?plain=1#L37)